networking.go

```diff
@@ -23,28 +23,15 @@ type FormDetails struct {
 	LocationUser string
 }
 
-type PingMsmt struct {
-	IP        string
-	PktSent   int
-	PktRecv   int
-	PktLoss   float64
-	MinRtt    float64
-	AvgRtt    float64
-	MaxRtt    float64
-	StdDevRtt float64
-}
-
 type Results struct {
-	UUID       string
-	IPaddr     string
-	Timestamp  string
-	IcmpPing   PingMsmt
-	MinIcmpRtt float64
+	UUID      string
+	Timestamp string
+	PingStats *ping.Statistics
```

Comment:
The [webpage results makes use of MinIcmpRtt](https://github.com/brave-experiments/proxy-detection-server/blob/8e460d16ffa70a29662df7ccf6706aa807737180/html_templates.go#L246) and needs to be changed too.